### ISSUE TITLE
Keeweb 1.4.17: passwordfile cannot be opened (suddenly) when on a network drive

### ISSUE THREAD

**Issue by Thomas-1985** [keeweb/keeweb#546]
Thanks for taking the time to contribute!

- I am using the windows app v1.4.17
-Suddenly (my passwordsafe is located on a network drive) i could not open it anymore, i get the error message attached
-The file named under appdata doesn't exist (i have offlinefiles disabled afterwards but this didn't help)
![keeweb](https://cloud.githubusercontent.com/assets/9249624/23455124/d61f5122-fe6e-11e6-9af3-394bfcc800be.jpg)
-Opening a file locally on my computer works however

Cheers

Thomy

**Comment by antelle** [comment]
I think, you should re-add your file to keeweb. Somehow your files from AppData disappeared.
> i have offlinefiles disabled afterwards 

What do you mean? We don't have an option to disable offline files.

**Comment by Thomas-1985** [comment]
Hi

Yeah thats what i tried, click on open and then navigate to the filer. After entering the password, the error message appears again, there is no way around rather than opening a file which is on the local filesystem.

I disabled the Windows offline files feature because i thought the error has to do with it but it doesn't

**Comment by antelle** [comment]
Have you deleted it from the list before clicking Open?

**Comment by Thomas-1985** [comment]
I just did this and i think it did the trick. Now i am able to open the file on the network-share again ;)

**Comment by antelle** [comment]
Perfect. So, somehow your file from AppData got deleted, but I don't know how this could happen...

**Comment by cleippi** [comment]
I'm experiencing this exact same issue. I'm pretty new to keeweb, but here's what I did:

1. Created the new password database
2. Sync'd it to google drive
3. Saved a copy locally (documents folder)
4. Closed the google drive version
5. Opened the local copy
6. Sync'd to dropbox
7. Closed the open database
8. Now from the main screen I try to open the file from dropbox and it's giving this error.

I'm really new to all of this, so I'm not sure what step is the problem, but that's the process I went through recently. As far as I can tell, it's a problem with the dropbox integration.
